### PR TITLE
Documented the new date_label and time_label options

### LINKED ISSUE #10292: [Form] Added options for separate date/time labels in DateTimeType
https://github.com/symfony/symfony/pull/21002

### PR DESCRIPTION
**Pull request by javiereguiluz**
Fixes #10292.

### CODE REVIEW

File: reference/forms/types/datetime.rst [hunk 1]
```diff
@@ -18,6 +18,7 @@ the data can be a ``DateTime`` object, a string, a timestamp or an array.
 | Options              | - `choice_translation_domain`_                                              |
 |                      | - `date_format`_                                                            |
 |                      | - `date_widget`_                                                            |
+|                      | - `date_label`_                                                             |
```
Comment (xabbuh) [inline reference/forms/types/datetime.rst]: we should sort the options IMO

File: reference/forms/types/datetime.rst [hunk 2]
```diff
@@ -29,6 +30,7 @@ the data can be a ``DateTime`` object, a string, a timestamp or an array.
 |                      | - `months`_                                                                 |
 |                      | - `seconds`_                                                                |
 |                      | - `time_widget`_                                                            |
+|                      | - `time_label`_                                                             |
```
Comment (xabbuh) [inline reference/forms/types/datetime.rst]: same here

Comment (xabbuh) [reply reference/forms/types/datetime.rst]: and we need to change the order below then too

File: reference/forms/types/datetime.rst [hunk 3]
```diff
@@ -72,6 +74,23 @@ date_widget
...
 
+date_label
+~~~~~~~~~~
+
+**type**: ``string`` | ``null`` **default**: The label is "guessed" from the field name
+
+.. versionadded:: 4.2
+    The ``date_label`` option was introduced in Symfony 4.2.
+
+Sets the label that will be used when rendering the date widget separately.
```
Comment (xabbuh) [inline reference/forms/types/datetime.rst]: I would remove the word "separately" here. It makes me understand the sentence as if the `date_label` option would only be used under certain circumstances.

File: reference/forms/types/datetime.rst [hunk 4]
```diff
@@ -151,6 +170,23 @@ time_widget
...
 
+time_label
+~~~~~~~~~~
+
+**type**: ``string`` | ``null`` **default**: The label is "guessed" from the field name
+
+.. versionadded:: 4.2
+    The ``time_label`` option was introduced in Symfony 4.2.
+
+Sets the label that will be used when rendering the time widget separately.
```
Comment (xabbuh) [inline reference/forms/types/datetime.rst]: same here

File: reference/forms/types/datetime.rst [hunk 5]
```diff
@@ -151,6 +170,23 @@ time_widget
...
+Setting to ``false`` will suppress the label::
```
Comment (xabbuh) [inline reference/forms/types/datetime.rst]: Setting it to [...]

File: reference/forms/types/datetime.rst [hunk 6]
```diff
@@ -72,6 +74,23 @@ date_widget
...
+date_label
+~~~~~~~~~~
+
+**type**: ``string`` | ``null`` **default**: The label is "guessed" from the field name
+
+.. versionadded:: 4.2
+    The ``date_label`` option was introduced in Symfony 4.2.
+
+Sets the label that will be used when rendering the date widget separately.
+Setting to ``false`` will suppress the label::
```
Comment (xabbuh) [inline reference/forms/types/datetime.rst]: Setting it to [...]

**Comment by javiereguiluz** [comment]
Fixed everything. Thanks for the review.

File: reference/forms/types/datetime.rst [hunk 7]
```diff
@@ -72,6 +74,23 @@ date_widget
 
 .. include:: /reference/forms/types/options/date_widget_description.rst.inc
 
+date_label
```
Comment (xabbuh) [inline reference/forms/types/datetime.rst]: looks like we need to move this up :)

**Pull request closed without merging**